packages/cycle-scripts/scripts/eject.js

```diff
@@ -29,21 +29,28 @@ inquirer.prompt([ejectConfirmation]).then(answers => {
   // STEP 1 - Prepare package.json
   // Declaring new scripts
   const scripts = {
-    start: 'node scripts/start.js',
-    test: 'node scripts/test.js',
-    build: 'node scripts/build.js'
+    start: 'NODE_ENV=development webpack-dev-server --config webpack.config.js',
+    build: 'NODE_ENV=production webpack --config webpack.config.js'
```

Comment:
This will lead to a different experience, no?  ie: no more `measureFileSizesBeforeBuild`